Model/Config.php

```diff
@@ -258,6 +258,16 @@ public function getOrderExportDisallowedCustomerGroups($websiteId = null)
         return explode(',', $this->getWebsiteData('order/export_disallowed_customer_groups', $websiteId));
     }
 
+    /**
+     * @param null|int|string|\Magento\Store\Model\Website $websiteId
+     *
+     * @return bool
+     */
+    public function isExportOrderPriceCurrency($websiteId = null)
```

Comment:
Instead of making a config setting, determine if we should send base or not base currency amounts by simply comparing `order_currency_code` and `store_currency_code` on order entity. If they are not equal - send not base amounts, if equal - send base amounts